File: lib/feature/cases/domain/entities/patient.dart
class Patient {
  final int? id;
  final String code;
  final String name;
  final int birthyear;
  final String gender;
  final String note;
  final DateTime? createdat;
  final DateTime? updatedat;

  Patient({
    this.id,
    required this.code,
    required this.name,
    required this.birthyear,
    required this.gender,
    required this.note,
    this.createdat,
    this.updatedat,
  });
}

File: ECG.Api/Models/Patient.cs
namespace ECG.Api.Models
{
    public class Patient
    {
        public int Id { get; set; }

        public string Code { get; set; } = null!;
        public string Name { get; set; } = null!;

        public DateOnly DateOfBirth { get; set; }

        public string Gender { get; set; } = null!;

        public string? Note { get; set; }

        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}
